commit.go

```diff
@@ -3,6 +3,8 @@ package dbft
 // Commit is an interface for dBFT Commit message.
 type Commit interface {
 	// Signature returns commit's signature field
-	// which is a block signature for the current epoch.
+	// which is a block signature for the current epoch (in case of dBFT 2.0)
+	// or a data CNs need to exchange with to move further to CommitAck phase
+	// (in case of anti-MEV dBFT version).
 	Signature() []byte
```

Comment:
Looks like this interface is a bit too specific, `VerifyCommit()` could handle the signature check and anything else needed. Signature check logic could be moved out of this library.

Reply:
> VerifyCommit() could handle the signature check and anything else needed.

No, it can't, because for decrypted tx parts verification at the PreCommit level PreBlock is needed, and thus this verification implemented as PreBlock.Verify callback, exactly as for Block.Verify and final block signature verification. And PreCommit.Verify should just perform various simple validation like marshalling correctness and etc.